homeassistant/components/advantage_air/climate.py

```diff
@@ -44,7 +44,7 @@
 ]
 
 ADVANTAGE_AIR_FAN_MODES = {
-    "auto": FAN_AUTO,
+    "autoAA": FAN_AUTO,
```

Comment:
is this supposed to have changed in this PR ?

Reply:
Yes, this correction is required as a result of the library changing from version 3.1 to 4.1.